shanoir-ng-datasets/src/main/java/org/shanoir/ng/vip/controller/ExecutionDataApiController.java

```diff
@@ -84,4 +131,115 @@ public ResponseEntity<?> getPath(
 
     }
 
+    @Override
+    public ResponseEntity<ExecutionDTO> createExecution(
+            @Parameter(name = "execution", required = true) @RequestBody final String executionAsString, @RequestHeader(HttpHeaders.AUTHORIZATION) String authenticationToken) throws EntityNotFoundException, SecurityException {
+
+        authenticationToken = authenticationToken.replace("Bearer ", "").trim();
```

Comment:
Eeeeh, getTokenUserName returns the user's username. We could get the authentication token, but I'm notreally sure it would be the same as the one we are sending in the API call

Reply:
Ah yes, i was thinking of getting directly the token through a public getJwtAuthenticationToken()